frontend/javascripts/oxalis/controller/scene_controller.ts

```diff
@@ -485,6 +488,7 @@ class SceneController {
 
   updateMeshesAccordingToLayerVisibility(): void {
     const state = Store.getState();
+    console.log("updating according to visible layers");
```

Comment:
remove

Reply:
oh sorry, I used that to confirm no unnecessary recomputation